src/pages/en/guides/imports.md

```diff
@@ -135,6 +137,42 @@ It can also be useful to place images in the `public/` folder as explained on th
 Adding **alt text** to `<img>` tags is encouraged for accessibility! Don't forget to add an `alt="a helpful description"` attribute to your image elements. You can just leave the attribute empty if the image is purely decorative.
 :::
 
+## `Astro.glob()`
+
+[`Astro.glob()`](/en/reference/api-reference/#astroglob) is a way to load many local files into your static site setup.
+
+`.glob()` only takes one parameter: a relative glob pattern of which local files you'd like to import. It’s asynchronous, and returns an array of the exports from matching files.
+
+```astro title="src/components/my-component.astro"
+---
+// returns an array of posts that live at ./src/pages/post/*.md
+const posts = await Astro.glob('../pages/post/*.md'); 
+---
+<!-- Returns an <article> for 5 blog posts -->
+<div>
+{posts.slice(0, 4).map((post) => (
+  <article>
+    <h1>{post.frontmatter.title}</h1>
+    <p>{post.frontmatter.description}</p>
+    <a href={post.frontmatter.url}>Read more</a>
+  </article>
+))}
+</div>
+```
+
+
```

Comment:
```suggestion
[`Astro.glob()`](/en/reference/api-reference/#astroglob) is a way to load many local files into your static site setup.

`Astro.glob()` only takes one parameter: a relative glob pattern matching the local files you'd like to import. It’s asynchronous, and returns an array of each matching file's exports.

```astro title="src/components/my-component.astro"
---
// imports all files that end with `.md` in `./src/pages/post/`
const posts = await Astro.glob('../pages/post/*.md'); 
---
<!-- Renders an <article> for the first 5 blog posts -->
<div>
{posts.slice(0, 4).map((post) => (
  <article>
    <h1>{post.frontmatter.title}</h1>
    <p>{post.frontmatter.description}</p>
    <a href={post.frontmatter.url}>Read more</a>
  </article>
))}
</div>
```

### Glob Patterns

A glob pattern is a file path that supports special placeholder characters. This lets it reference multiple files. For example,

`./pages/**/*.{md,mdx}` starts within the pages subdirectory of the current directory (`./pages`), looks through all of its subdirectories (`/**`), and checks for any filename (`/*`) that ends in either `.md` or `.mdx` (`.{md,mdx}`). 

[Read more about the glob pattern syntax](https://github.com/mrmlnc/fast-glob#pattern-syntax).
```